Source/UI/Coordinators/PasswordCoordinator.swift

```diff
@@ -308,27 +308,35 @@ extension PasswordCoordinator {
             completion()
         } else {
             Settings.setValue(true, forKey: hasLoggedInBeforeSettingsKey)
-            let viewModel = PasswordViewModel(
-                identifier: identifier,
-                loginFlowVariant: loginFlowVariant,
-                localizationBundle: self.configuration.localizationBundle
-            )
-            let message = viewModel.biometricsOnboardingMessage
-                .replacingOccurrences(of: "$0", with: configuration.appName)
-            let title = viewModel.biometricsOnboardingTitle
+            let context = LAContext()
+            context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil)
```

Comment:
Why is this called? The return value isn't used for anything here.

Reply:
https://developer.apple.com/documentation/localauthentication/lacontext/2867583-biometrytype
> This property is set only after you call the canEvaluatePolicy(_:error:) method, and is set no matter what the call returns. The default value is LABiometryType.none.

Reply:
Although might be extracted to a `var` so it will be clear these 3 lines are together required to get `biometryType`'s value.
```
    private var biometryType: LABiometryType {
        let context = LAContext()
        // `biometryType` property is set only after you call the canEvaluatePolicy(_:error:) method, and is set no matter what the call returns. The default value is LABiometryType.none.
        context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil)
        return context.biometryType
    }
```